.github/workflows/bench.yml

```diff
@@ -21,10 +21,18 @@ jobs:
         with:
           submodules: recursive
       - uses: dtolnay/rust-toolchain@stable
+
+      # Install the benchmark runner
+      - name: Requirements
+        run: cargo install --version 0.10.2 iai-callgrind-runner
+
+      # Create a file to write in beforehand
+      - name: Chore
+        run: sudo mkdir target/iai/honeycomb-utils/ && sudo touch target/iai/honeycomb-utils/bench.out
       
       # Run all benchmarks
-      - name: Run
-        run: cargo bench --benches
+      - name: Run 
+        run: cargo bench --bench prof-twomap-editing --bench prof-twomap-reading --bench prof-twomap-sewing-unsewing -- --save-summary=json >> target/iai/honeycomb-utils/bench.out
```

Comment:
I do not understand the append to bench.out if just created before ? You can simply redirect or use tee and you do not have to create files before hand.

Reply:
changes: bdaa301bc6efeba43b66854268f95f426d8ad2d6